specs/phase0/beacon-chain.md

```diff
@@ -183,6 +183,7 @@ The following values are (non-configurable) constants used throughout the specif
 
 | Name | Value |
 | - | - |
+| `ETH1_FOLLOW_DISTANCE` | `2**10` (= 1,024) |
```

Comment:
Now we are not expressing anymore that the distance is a distance of blocks. But I think it's clear enough from its use in the spec. Maybe with the descriptions PR we can add more comments for each constant.

Reply:
Agreed. I wasn't too happy with how it looks here.
I also considered just making the eth2 follow distance purely in seconds and getting rid of this constant but (1) you miss the meaning of ~1024 block  and (2) I'd rather leave the constants stable for implementers